src/hooks/useActionCenterSubscribers/useSendActionSubscriber.tsx

```diff
@@ -0,0 +1,86 @@
+import { usePrevious } from '@chakra-ui/react'
+import { fromAccountId } from '@shapeshiftoss/caip'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { useEffect } from 'react'
+
+import { useNotificationToast } from '../useNotificationToast'
+
+import { useActionCenterContext } from '@/components/Layout/Header/ActionCenter/ActionCenterContext'
+import { SendNotification } from '@/components/Layout/Header/ActionCenter/components/Notifications/SendNotification'
+import { actionSlice } from '@/state/slices/actionSlice/actionSlice'
+import { selectPendingWalletSendActions } from '@/state/slices/actionSlice/selectors'
+import { ActionStatus } from '@/state/slices/actionSlice/types'
+import { selectTxs } from '@/state/slices/selectors'
+import { serializeTxIndex } from '@/state/slices/txHistorySlice/utils'
+import { useAppDispatch, useAppSelector } from '@/state/store'
+
+export const useSendActionSubscriber = () => {
+  const { isDrawerOpen, openActionCenter } = useActionCenterContext()
+
+  const dispatch = useAppDispatch()
+
+  const toast = useNotificationToast({ duration: isDrawerOpen ? 5000 : null })
+
+  const pendingSendActions = useAppSelector(selectPendingWalletSendActions)
+  const txs = useAppSelector(selectTxs)
+
+  useEffect(() => {
+    pendingSendActions.forEach(action => {
+      const { accountId, txHash } = action.transactionMetadata
+      const accountAddress = fromAccountId(accountId).account
+
+      const serializedTxIndex = serializeTxIndex(accountId, txHash, accountAddress)
+      const tx = txs[serializedTxIndex]
+
+      if (!tx) return
+      if (tx.status !== TxStatus.Confirmed) return
+
+      dispatch(
+        actionSlice.actions.upsertAction({
+          ...action,
+          status: ActionStatus.Complete,
+          updatedAt: Date.now(),
+          transactionMetadata: {
+            ...action.transactionMetadata,
+            message: 'modals.send.youHaveSent',
+          },
+        }),
+      )
+
+      const isActive = toast.isActive(txHash)
+
+      // No double-toasty
+      if (isActive) return
```

Comment:
i.e since the notification itself may also live-update itself, and *may* already be handling success update (assuming it's opened, hence live-updating) then failure to do so means we'd have double notifications.